src/Models/View.php

```diff
@@ -1,12 +1,38 @@
 <?php
```

Comment:
I'd like to keep this file as it was, as well as removing the IDE helper. It's overkill to me.